bindata/manifests/cni-config/ovs/ovs-cni-config.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: {{.NetworkName}}
+  namespace: {{.NetworkNamespace}}
+  annotations:
+    k8s.v1.cni.cncf.io/resourceName: {{.CniResourceName}}
+spec:
+  config: '{
+  "cniVersion":"0.3.1",
```

Comment:
please update this one version to also 1.0.0

Reply:
Hey, yury on PTO - ill address it in followup

Reply:
since yury is on PTO i addressed this in #696